docs/client-libraries-cpp-initialize.md

```diff
@@ -0,0 +1,7 @@
+---
+id: client-libraries-cpp-initialize
+title: Initialize a C++ client object
+sidebar_label: "Initialize"
+---
+
+Coming soon...
```

Comment:
Generally, I'd avoid "Coming soon" before we do have the content. I still remember the history that the release timeline coming soon for months and finally getting removed.

For user docs, when you add a file, the URL is exported and it's more complicated if it's finally given up and do the cleanup job.

Reply:
Thanks for the suggestion. As we talked, it's kind of a placeholder page for the next move of [PIP-228](https://github.com/apache/pulsar/issues/18822) implementation. @RobertIndie is going to add details to these "coming soon" topics in the coming weeks of Q1. That's the collaborative way we come up with:) And we'll make sure they are available before the release of the next major version.

Reply:
Sounds good :)

While making a "Comming soon" mark doesn't help compared with directly pushing the content when it's finished, only increases the risk that if we cannot deliver it on time it exports blank pages for users. I'd suggest we avoid such a pattern the next time.